demos/pet-shop/src/app.js

```diff
@@ -1,113 +1,117 @@
 
 let App = {
   web3: null,
   contracts: {},
 
-  init: async function() {
+  init: async function () {
     // Load pets.
-    $.getJSON("pets.json", function(data) {
-      var petsRow = $("#petsRow");
-      var petTemplate = $("#petTemplate");
+    $.getJSON('pets.json', function (data) {
+      var petsRow = $('#petsRow')
+      var petTemplate = $('#petTemplate')
 
       for (let i = 0; i < data.length; i++) {
-        petTemplate.find(".panel-title").text(data[i].name);
-        petTemplate.find("img").attr("src", data[i].picture);
-        petTemplate.find(".pet-breed").text(data[i].breed);
-        petTemplate.find(".pet-age").text(data[i].age);
-        petTemplate.find(".pet-location").text(data[i].location);
-        petTemplate.find(".btn-adopt").attr("data-id", data[i].id);
-
-        petsRow.append(petTemplate.html());
+        petTemplate.find('.panel-title').text(data[i].name)
+        petTemplate.find('img').attr('src', data[i].picture)
+        petTemplate.find('.pet-breed').text(data[i].breed)
+        petTemplate.find('.pet-age').text(data[i].age)
+        petTemplate.find('.pet-location').text(data[i].location)
+        petTemplate.find('.btn-adopt').attr('data-id', data[i].id)
+
+        petsRow.append(petTemplate.html())
       }
-    });
+    })
 
-    return await App.initWeb3();
+    return await App.initWeb3()
   },
 
-  initWeb3: async function() {
+  initWeb3: async function () {
     // Modern dapp browsers...
-    let web3Provider = null;
+    let web3Provider = null
     if (window.ethereum) {
-      web3Provider = window.ethereum;
+      web3Provider = window.ethereum
       try {
         // Request account access
-        await window.ethereum.enable();
+        await window.ethereum.enable()
       } catch (error) {
         // User denied account access...
-        console.error("User denied account access");
+        console.error('User denied account access')
       }
     }
     // Legacy dapp browsers...
     else if (window.web3) {
-      web3Provider = window.web3.currentProvider;
+      web3Provider = window.web3.currentProvider
     }
     // If no injected web3 instance is detected, fall back to Ganache
     else {
-      web3Provider = new Web3.providers.HttpProvider("http://localhost:7545");
+      web3Provider = new Web3.providers.HttpProvider('http://localhost:7545')
     }
 
-    let provider = ArbProvider("http://localhost:1235", web3Provider);
-    App.web3 = new Web3(provider); // eslint-disable-line require-atomic-updates
+    let provider = ArbProvider(
+      'http://localhost:1235',
+      web3Provider,
+      'http://localhost:1237'
+    )
+    App.web3 = new Web3(provider) // eslint-disable-line require-atomic-updates
 
-    return App.initContract();
+    return App.initContract()
   },
 
-  initContract: function() {
-    let adoption = require("../build/contracts/Adoption.json");
+  initContract: function () {
+    let adoption = require('../build/contracts/Adoption.json')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['adoption' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)